cleverpush/src/main/java/com/cleverpush/banner/AppBannerModule.java

```diff
@@ -355,61 +356,65 @@ private boolean appVersionFilter(boolean allowed, Banner banner) {
         try {
             PackageInfo pInfo = this.getCurrentActivity().getPackageManager().getPackageInfo(this.getCurrentActivity().getPackageName(), 0);
             String versionName = pInfo.versionName;
-            return this.checkRelationFilter(allowed, banner.getBannerAppVersionFilterRelation(), versionName, banner.getAppVersionFilterValue());
+            return this.checkRelationFilter(allowed, banner.getBannerAppVersionFilterRelation(), versionName, banner.getAppVersionFilterValue(), banner.getFromVersion(), banner.getToVersion());
         } catch (Exception e) {
             e.printStackTrace();
             Logger.e(TAG, "Error checking app version filter", e);
         }
+
         return allowed;
     }
 
-    private boolean checkRelationFilter(boolean allowed, CheckFilterRelation relation, String versionName, String compareValue) {
+    private boolean checkRelationFilter(boolean allowed, CheckFilterRelation relation, String appVersion, String compareValue, String fromVersion, String toVersion) {
         VersionComparator vc = new VersionComparator();
```

Comment:
not sure if we want to shorten this @gbhrdt

Reply:
here only added fromVersion and toVersion from my end